src/Chapter3.hs

```diff
@@ -480,6 +546,81 @@ After defining the city, implement the following functions:
    and at least 10 living __people__ inside in all houses of the city in total.
 -}
 
+data Person 
+    = Person 
```

Comment:
This data type actually doesn't bring any weight to your system. Do you think that you can redesign the solution a bit without empty data types?